x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/discover/search_source_alert.ts

```diff
@@ -575,6 +576,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       await testSubjects.click('saveRuleButton');
 
+      await retry.waitFor('consumer selection modal', async () => {
+        return await testSubjects.exists('confirmModalConfirmButton');
+      });
+
+      await testSubjects.click('confirmModalConfirmButton');
+
```

Comment:
I wonder why it needs a second confirmation

Reply:
The first confirmation is for the new consumer selector modal (which this PR adds), and the second confirmation is for the existing "no actions" confirmation modal.